config/features.yml

```diff
@@ -49,10 +49,6 @@ features:
   hca_use_facilities_API:
     actor_type: user
     description: Allow list of medical care facilites to be fetched by way of the Facilities API.
-  hca_vaprofile_military_info:
-    actor_type: user
-    description: Enables prefilling 1010ez using VA profile military data
-    enable_in_development: false
```

Comment:
Removing per this conversation: https://dsva.slack.com/archives/CMJ2V70UV/p1692296292834139?thread_ts=1692294670.095529&cid=CMJ2V70UV